tools/recetox_aplcms/macros.xml

```diff
@@ -7,6 +7,17 @@
         </requirements>
     </xml>
 
+    <xml name="annotation">
+       <edam_topics>
+              <edam_topic>topic_3172</edam_topic>
+              <edam_topic>topic_0091</edam_topic>
+              <edam_topic>topic_3520</edam_topic>       
+       </edam_topics>
+       <xrefs>
+              <xref type="bio.tools">recetox-aplcms</xref>

```

Comment:
IFF you have a bio.tools ID and the EDAM operations are stored under the bio.tools ID.

Galaxy is smart enought to annotate your tools automatically with EDAM using the bio.tools ID - means you do NOT need to add edam_topics ONLY the bio.tools ID.

Reply:
Oh nice - I didn't know that!

What would be the case if a tool has multiple steps and we want to be precise? Like some steps in recetox-aplcms are peak detection and some are filtering - can we be explicit about that?

Reply:
Also would be great if the documentation on bio.tools and EDAM and the Galaxy integration could be updated, or at least I didn't find this information there: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#edam-topics-and-operations

Reply:
> Oh nice - I didn't know that!
> 
> What would be the case if a tool has multiple steps and we want to be precise? Like some steps in recetox-aplcms are peak detection and some are filtering - can we be explicit about that?

Ideally, you would capture that in bio.tools. So I guess extra entries for every step? If this is unpractical, then what you have here works and will overwrite the bio.tools given operations.

Reply:
Sure, go for it. And maybe its different/easier for other tools.